src/index.ts

```diff
@@ -239,7 +233,7 @@ DNS.prototype.createZone = function(name, config, callback) {
 DNS.prototype.getZones = function(query, callback) {
   var self = this;
 
-  if (is.fn(query)) {
+  if (typeof query === 'function') {
```

Comment:
grumble: This change was not really necessary as part of the TS conversion. However, since you have done it, the implementation of the two sides is not identical, which makes this theoretically semver major. But this PR is semver major anyway, so 🤷‍♂️? If you were going to do this, I would prefer if you were to remove the dependency on `is` completely rather than a half hearted change here.